shared/ui/src/components/ElementsBox/elementsBox.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { ElementsBox } from '.';
+import styled from '@emotion/styled';
+
+export default {
+  title: 'ElementsBox',
+  component: ElementsBox,
+  argTypes: {},
+} as ComponentMeta<typeof ElementsBox>;
+
+const Template: ComponentStory<typeof ElementsBox> = (args) => (
+  <ElementsBox {...args} />
+);
+
+const cel1 = () => <El1 />;
+const cel2 = () => <El2 />;
+const cel3 = () => <El3 />;
```

Comment:
컴포넌트의 예시로 쓰고싶으면 변수의 첫글자를 대문자로!!
```suggestion
const Cel1 = () => <El1 />;
const Cel2 = () => <El2 />;
const Cel3 = () => <El3 />;
```